src/views/Dashboard/Lab/Registration/Certification.vue

```diff
@@ -155,7 +175,10 @@ export default {
     certificationDialog: false,
     isUploading: false,
     isEditCertificationDialog: false,
-    files: []
+    files: [],
+    sertifDialog: false,
```

Comment:
please use proper english for naming, so that it could be easily understand with others and in the future

Reply:
sorry for bad english, i already change naming with proper english.